azure-pipelines.yml

```diff
@@ -0,0 +1,48 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - main
+      - release/*
+
+resources:
+  repositories:
+    - repository: niveristand-custom-device-build-tools
+      type:       github
+      ref:        main
+      endpoint:   ni (2)
+      name:       ni/niveristand-custom-device-build-tools
+# Pipeline Completion Triggers:
+  pipelines:
+    - pipeline: VeristandMessageLibrary
+      source:   ni.niveristand-custom-device-message-library
+      trigger:  true
+
+stages:
+  - template: azure-templates/stages.yml@niveristand-custom-device-build-tools
+    parameters:
+      lvVersionsToBuild: 
+        - version: '2020'
+          bitness: '32bit'
+        - version: '2021'
+          bitness: '64bit'
+        - version: '2023'
+          bitness: '64bit'
+
+      buildSteps:
+        - projectLocation: 'Source\Message Library.lvproj'
+          buildOperation: 'ExecuteAllBuildSpecs'
+          target: 'All'
+          buildSpec: 'All'
+
+      releaseVersion: '23.0.0'
+      quarterlyReleaseVersion: '2023 Q1'
+      buildOutputLocation: 'Built'
+      archiveLocation: '\\daqserver\temp\papower\Measurements\VeriStandAddons\messaging_library'
```

Comment:
I think we can update this to dump to a semi-permanent location.

How about `\\nirvana\measurements\VeriStandAddons\prototype\messaging_library`? And we can delete the entire `prototype` directory when we're done?

Reply:
resolved in [02cedfe](https://github.com/ni/niveristand-custom-device-message-library/pull/18/commits/02cedfe9d69dbafe5ac37d61ab28b403d45d0f17)